src/main/java/hello/AppConfiguration.java

```diff
@@ -1,26 +0,0 @@
-package hello;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-/**
- * Created by Vladyslav Usenko on 16.01.2016.
- */
-@Configuration
-public class AppConfiguration {
-
-    @Bean
-    public TaskRepository taskRepository() {
```

Comment:
There's no need in creating TaskRepository with configuration, it will be anyway created with classpath scan